_drafts/restic-foundation1-cdc.md

```diff
@@ -0,0 +1,268 @@
+---
+layout: post
+title: Foundation - Introducing Content Defined Chunking (CDC)
+---
+
+This post will explain Content Defined Chunking (CDC) and how it is used by
+restic.
+
+Backup programs need to deal with large volumes of changing data. Saving the
+whole copy of each file again to the backup location when a subsequent (usually
+called "incremental") backup is created is not efficient. Over time, different
+strategies have emerged to handle data in such a case.
+
+In a backup program, data de-duplication can be applied in two locations:
+Removing duplicate data from the same or different files within the same backup
+process (*inter-file de-duplication*), e.g. the initial backup, or removing it
+between several backups of the same data (*inter-backup de-duplication*). While
+the former is desirable to have, the latter is much more important.
+
+### Strategies
+
+The most basic strategy is to only save files that have changed since the last
+backup, this is where the term "incremental" backup comes from. This way,
+unmodified files are not stored again on subsequent backups. But what happens if
+just a small portion of a large file is modified? Using this strategy, the
+modified file will be saved again, although most of it did not change.
+
+A better idea is splitting files into smaller fixed-size pieces (called
```

Comment:
nitpick: maybe "A better idea is to split files ..."? I think that's a bit more clear...